runtime/server_http_request.go

```diff
@@ -130,6 +130,11 @@ func (req *ServerHTTPRequest) Context() context.Context {
 	return req.httpRequest.Context()
 }
 
+// StartTime returns the request's start time.
```

Comment:
Where is this used?  I am not seeing it in the diff

Reply:
This is from a previous diff, somehow got added here. I'll remove it.